### ISSUE TITLE
Research for Fundus

### ISSUE THREAD

**Issue by nadjaweisskopf** [leihs/leihs#156]
Please gather the following information for Fundus: 

between 15.10.16 and today (pool Fundus)
- average of items per singed and closed contracts since 15.10.16
- number of items lended since 15.10.16
- total amount (value list) of all items lended since 15.10.16

**Comment by nimaai** [comment]
@nadjaweisskopf **nur** items and **no** options, right?

**Comment by nadjaweisskopf** [comment]
No, sorry, Options too. They mostly work with options.

**Comment by nimaai** [comment]
between 15.10.16 and today (pool Fundus)
* average of items per singed and closed contracts since 15.10.16
```ruby
ip = InventoryPool.all.find { |ip| ip.name == "Fundus-TdK" }
contracts = ip.reservations_bundles.where(status: [:signed, :closed]).select { |c| c.created_at >= Date.new(2016,10,15) }
items_sum = contracts.map(&:item_lines).map(&:count).sum
options_sum = contracts.map { |c| c.option_lines.map(&:quantity) }.flatten.sum
average = ((items_sum + options_sum) / contracts.count)

=> 12
```
